### PR TITLE
Contact Us Api

### PR DESCRIPTION
**Pull request by manojkgorle**

### CODE REVIEW

**Comment by manojkgorle** [comment]
Contact us page conflicts are resolved.
While in production delete tokens.json & generate new OAuth tokens from cloud console

File: client/api/contact_us/credentials.json
Comment (Arush-Bansal) [inline client/api/contact_us/credentials.json:1]: clinet secret should be in environment variables

Comment (Arush-Bansal) [inline client/api/contact_us/credentials.json:1]: Client secret should be in environment variables

File: client/api/contact_us/index.js
Comment (Arush-Bansal) [inline client/api/contact_us/index.js:1]: please change the following

  subject: `Thank you for reaching out to the Entrepreneurship Development Cell, IIT Delhi`,
  html: `
    Dear ${targetName},<br><br>
    Thank you for contacting the Entrepreneurship Development Cell at IIT Delhi. We appreciate your interest and inquiry.<br><br>
    Our team is dedicated to supporting and promoting entrepreneurship, and we are delighted that you've taken the first step to connect with us.<br><br>
    Rest assured, we have received your message, and one of our team members will get back to you within a week. In the meantime.<br><br>
    We look forward to assisting you in your entrepreneurial journey.<br><br>
    Best regards,<br>
    Entrepreneurship Development Cell<br>
    Indian Institute of Technology (IIT) Delhi<br>
  `,

**Review by Arush-Bansal — CHANGES_REQUESTED** [review]

File: client/api/contact_us/credentials.json
Comment (Arush-Bansal) [reply client/api/contact_us/credentials.json:1]: client_id also

**Pull request closed without merging**